File: app/src/main/java/com/dtu/capstone2/ereadingandroid/network/request/AccountLoginRequest.java
package com.dtu.capstone2.ereadingandroid.network.request;

import lombok.AllArgsConstructor;
import lombok.Data;

/**
 * Create by Nguyen Van Phuc on 3/11/19
 */
@AllArgsConstructor
@Data
public class AccountLoginRequest {
    private String userName;
    private String passwork;
}

File: app/src/main/java/com/dtu/capstone2/ereadingandroid/network/request/AccountLoginRequest.kt
package com.dtu.capstone2.ereadingandroid.network.request

import com.google.gson.annotations.SerializedName
import java.util.*

/**
 * Create by Nguyen Van Phuc on 3/11/19
 */
data class AccountLoginRequest(@SerializedName("username") val userName: String,
                               @SerializedName("password") val password: String)



